src/components/ProfilePicture.tsx

```diff
@@ -0,0 +1,22 @@
+import Image from "next/image";
+import demo from "../../public/profile.jpg";
+
+const ProfilePicture = ({ from } : { from: string }) => {
+  let dimensions = 0;
+
+  if (from === "likes") {
+    dimensions = 15;
+  } else if (from === "blog") {
+    dimensions = 16;
+  } else if (from === "profile") {
+    dimensions = 30;
+  }
```

Comment:
❌ 
I don't like this `from` prop, it makes the code more complex, imagine that we have two places that have exactly the same image dimensions, we will end up with two if statements with the same value, this is not DRY

also what if we added this component to a new place, each time we want to use this somewhere we will have to create a new if statement and that means we need to change the code every time we want to use this. this is not OCP, 

instead, I will add something more generic, like add the ability to pass the `dimensions` as a prop
or maybe even better I will change the `from` to `size` and I will have a few sizes like this [xs, sm, md, lg, xl, xxl] and each one will have it is own dimensions, can you see what I am trying to say?